server/utils/nunjucksSetup.ts

```diff
@@ -85,7 +85,9 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
     njkEnv.addGlobal('tagManagerUrl', `https://www.googletagmanager.com/ns.html?id=${tagManagerId.trim()}`)
   }
 
-  njkEnv.addGlobal('dateFieldValues', dateFieldValues)
+  njkEnv.addGlobal('dateFieldValues', function sendContextToDateFieldValues(fieldName: string, errors: ErrorMessages) {
+    return dateFieldValues(fieldName, this.ctx, errors)
+  })
```

Comment:
it would probably be ideal if this change were accompanied by a test which demonstrates this fixed behaviour. i.e. checking out this commit and running that test with lines 89-90 commented out would fail...